### PR TITLE
docs: Added managed flow required configuration

### PR DESCRIPTION
**Pull request by jasuno**
# Overview

  

# Test Plan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,23 @@ If your project is using [Expo](https://expo.io/), install the library and the c
 expo install react-native-modal-datetime-picker @react-native-community/datetimepicker
 ```
 
+You will also need to configure supported appearance styles in your app.json:
```
Comment (mmazzarolo) [inline README.md:39]:
```suggestion
You can configure appearance styles in your `app.json`:
```

Comment (mmazzarolo) [reply README.md:39]: It's not a requirement... right?

**Review by mmazzarolo — CHANGES_REQUESTED** [review]

**Comment by mmazzarolo** [comment]
Thanks :)

File: README.md (hunk 1 above)
Comment (mmazzarolo) [reply README.md:39]: Also, I don't think the iOS interface can be customized until #549 lands, right?

Comment (jasuno) [reply README.md:39]: So I just implemented "react-native-modal-datetime-picker" in my project, it seems to be a requirement, if that config is not set up then the system color schema always returns "light".

Comment (mmazzarolo) [reply README.md:39]: Gotcha, I updated the suggestion :)

Comment (jasuno) [reply README.md:39]: Yeah, that's correct, if the "userInterfaceStyle": "automatic" config is not set up and the user has their device in dark mode what happens is that it looks like the modal is empty although it is actually not, it's white text on white background, the config lets your code understand what colour scheme the system is using. This config is not required for the modal to work, but I spent a few hours looking for why I wasn't able to see it and the config was the solution so I thought I would add it to the documentation so others didn't have to do the same.

Comment (mmazzarolo) [reply README.md:39]: Could you remove the `android` and `ios` bits so that we can merge it (as they're not needed)?

Comment (jasuno) [reply README.md]: Changes were pushed

**Pull request merged**